### DISCUSSION TITLE
Welcome to home-assistant-comfoconnect Discussions!

### DISCUSSION THREAD

**Discussion by michaelarnauts** [michaelarnauts/home-assistant-comfoconnect#20] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by LancevdMeer** [comment]
Re: LAN-C IP Adress not recognized

Hi Michael + Community

Thank you for the great work done. I followed all instructions by the letter (HA vm version, on a MAC with HACS), and used HACS to get the repository added. However - the last step is failing :

 when I want to add the integration,I get “invalid host” when entering the LAN-C IP address

As you can see from my router screenshot, I am using the correct IP (the MAC address matches that printed on the LAN-C

I have factory reset my LAN-C to make sure the gateway is on default 0000, and am able though the iOS comfocontrol app to access my ComfoAirQ450 from external internet (eg from 4G eg outside my network)

My home-assist install has no issues to connect with other devices on my lan network nor on my wifi. The Mac being used is lan-connected to my router. 

Nevertheless, I keep getting  “invalid host” 

Please advice

![IMG_7579](https://github.com/michaelarnauts/home-assistant-comfoconnect/assets/42781470/3c3bb008-64c8-4b27-b0b6-bfaa3377ad5a)
![IMG_7578](https://github.com/michaelarnauts/home-assistant-comfoconnect/assets/42781470/7428d1ae-e217-44c0-b4e2-6dd07a5ae0eb)

**Comment by jormarmon** [comment]
Hello,

I have the same problem.

Uninstalled from HACS and returned to pycomfoconnect.

**Comment by kaiorurulad** [comment]
Hi Michael + Community,
I have it installed and working.
How do I create an Automation to turn the fan on to boost for 20 minutes please?

**Reply by michaelarnauts** [reply]
Create an automation to:
* Set speed to high
* Wait 20 minutes
* Set speed to low

**Reply by kaiorurulad** [reply]
That works well thank you.
